OMEdit/OMEditGUI/Editors/TransformationsEditor.cpp

```diff
@@ -51,6 +51,25 @@ TransformationsEditor::TransformationsEditor(TransformationsWidget *pTransformat
 }
 
 /*!
+ * \brief TransformationsEditor::popUpCompleter()
+ * \we do not have completer for this
+ */
+void TransformationsEditor::popUpCompleter()
+{
+
+}
+
+QStringList TransformationsEditor::getKeywords()
```

Comment:
Why does `TransformationsEditor` still have `getKeywords()` and `getTypes()` functions?